docs/source/api/module.md

```diff
@@ -2,8 +2,146 @@ torchsynth.module
 =================
 
 ```{eval-rst}
-.. automodule:: torchsynth.module
-   :members:
-   :undoc-members:
-   :show-inheritance:
+SynthModule
+-----------
+.. autoclass:: torchsynth.module.SynthModule
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :exclude-members: default_parameter_ranges
+
+Audio Rate Modules
+------------------
+These modules operate at full audio sampling rate.
+
+AudioMixer
+==========
+.. autoclass:: torchsynth.module.AudioMixer
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+ControlRateUpsample
+===================
+.. autoclass:: torchsynth.module.ControlRateUpsample
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+Noise
+=====
+.. autoclass:: torchsynth.module.Noise
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+SineVCO
+=======
+.. autoclass:: torchsynth.module.SineVCO
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+SquareSawVCO
+============
+.. autoclass:: torchsynth.module.SquareSawVCO
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+TorchFmVCO
+==========
+.. autoclass:: torchsynth.module.TorchFmVCO
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+VCA
+===
+.. autoclass:: torchsynth.module.VCA
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+VCO
+===
+.. autoclass:: torchsynth.module.VCO
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :private-members: _forward
```

Comment:
One thing I'm not sure about yet is how to document the forward for each of our synthmodules. Since the args are defined in `_forward` -- but we don't want users to call `_forward` directly ever. If anything they should use `__call__`

Reply:
@jorshi rename it to `__forward` and then it can't really be called externally?

Reply:
IDK, maybe we need to have a quick chat?